### PR TITLE
feat(ingest): hide cartesian product warnings in GE profiler

### PR DESCRIPTION
**Pull request by hsheth2**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/utilities/sqlalchemy_query_combiner.py [hunk 1]
```diff
@@ -333,7 +340,10 @@ def _execute_queue(self, main_greenlet: greenlet.greenlet) -> None:
             # Extract the results into a result for each query.
             index = 0
             for _, query_future in pending_queue.items():
-                cols = query_future.query.columns
+                query = query_future.query
+                if IS_SQLALCHEMY_1_4:
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/utilities/sqlalchemy_query_combiner.py:344]: Why is this part necessary? The subquery piece, whereas previously it was not

**Review by jjoyce0510 — APPROVED** [review]
One minor question

File: metadata-ingestion/src/datahub/utilities/sqlalchemy_query_combiner.py (hunk 1 above)
Comment (hsheth2) [reply metadata-ingestion/src/datahub/utilities/sqlalchemy_query_combiner.py:344]: not necessary - just suppresses a different warning that SA produces

**Comment by treff7es** [comment]
@hsheth2 there is a linter error in the pr
